dpd/src/api_server.rs

```diff
@@ -3061,6 +3070,200 @@ async fn counter_get(
         .map_err(HttpError::from)
 }
 
+#[derive(Deserialize, Serialize, JsonSchema)]
+pub struct MulticastGroupIdParam {
+    pub group_id: String,
+}
+
+// Implement conversion to mcast::Identifier
+impl MulticastGroupIdParam {
```

Comment:
This feels like it wants to be `impl TryFrom<MulticastGroupIdParam> for mcast::Identifier`

Reply:
yep.